digiroad2-oracle/src/main/resources/db/migration/V236_create_new_roawday_asset.sql

```diff
@@ -0,0 +1,441 @@
+VALUES (480, 'K4 Varoitusviiva', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (490, 'K5 Sulkualue', 'linear', 'db_migration_v236');
+INSERT INTO ASSET_TYPE (ID, NAME, GEOMETRY_TYPE, CREATED_BY)
+VALUES (500, 'K6 Reunaviiva', 'linear', 'db_migration_v236');
+
+-- LOCALIZED_STRING VALUE
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K1 keskiviiva', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K2 ajokaistaviiva', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K3 Sulkuviiva', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K4 Varoitusviiva', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K5 Sulkualue', 'db_migration_v236', SYSDATE);
+INSERT INTO LOCALIZED_STRING (ID, VALUE_FI, CREATED_BY, CREATED_DATE)
+VALUES (primary_key_seq.nextval, 'K6 Reunaviiva', 'db_migration_v236', SYSDATE);
+
+-- PROPERTY VALUE
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'text', 1, 'db_migration_v236', 'center_line',
+        (SELECT ID FROM LOCALIZED_STRING WHERE VALUE_FI = 'K1 keskiviiva'));
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 460, 'text', 1, 'db_migration_v236', 'lane_line',
+        (SELECT ID FROM LOCALIZED_STRING WHERE VALUE_FI = 'K2 ajokaistaviiva'));
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 470, 'text', 1, 'db_migration_v236', 'barrier_line',
+        (SELECT ID FROM LOCALIZED_STRING WHERE VALUE_FI = 'K3 Sulkuviiva'));
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 480, 'text', 1, 'db_migration_v236', 'warning_line',
+        (SELECT ID FROM LOCALIZED_STRING WHERE VALUE_FI = 'K4 Varoitusviiva'));
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 490, 'text', 1, 'db_migration_v236', 'closed_area',
+        (SELECT ID FROM LOCALIZED_STRING WHERE VALUE_FI = 'K5 Sulkualue'));
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 500, 'text', 1, 'db_migration_v236', 'edge_line',
+        (SELECT ID FROM LOCALIZED_STRING WHERE VALUE_FI = 'K6 Reunaviiva'));
```

Comment:
Was created one property per regulation numbers values. Those values it's to be stored as an ENUMERATED_VALUE.

Because the property will be only one, will be a property related to the fields "Regulation Number", and then we will create the multiple ENUMERATED_VALUES.

Example:
```SQL
INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED,CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
	VALUES (primary_key_seq.nextval, 460, 'single_choice', 1,'db_migration_v236', 'lengthOfRoadAxisMarking_regulation_number', (SELECT ID FROM LOCALIZED_STRING WHERE VALUE_FI = 'Name for this in Finnish'));


-- ENUMERATED VALUES
INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
	VALUES (primary_key_seq.nextval, 1,'K1 keskiviiva','','db_migration_v236',(SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'lengthOfRoadAxisMarking_regulation_number'));

Rest of ENUMERATED_VALUES for this property......